promxy.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: promxy
+  version: 0.0.90
+  epoch: 0
+  description: An aggregating proxy to enable HA prometheus.
+  copyright:
+    - license: MIT
```

Comment:
I was doing a final check and noticed that upstream image is also pulling in ca-certificates-bundle in the final image. 
Should we also include ca-certificates-bundle as runtime dependency here looking at upstream image config? https://github.com/jacksontj/promxy/blob/a622d5348e93c81d4e929ddb12a8bf6d25199337/Dockerfile#L16

Reply:
Good point. I'll add it as:

```
package:
  # ...
  dependencies:
    runtime:
      - ca-certificates-bundle
```